### ISSUE TITLE
Support placeholder attribute for ZoomableAsyncImage

### ISSUE THREAD

**Issue by minas1** [saket/telephoto#28]
Coil's `AsyncImage` has a placeholder attribute: `placeholder: Painter? = null`.

Please add support for it in `ZoomableAsyncImage`.

-----

In the [docs](https://saket.github.io/telephoto/zoomableimage/#placeholders) there's mention of using placeholders using `placeholderMemoryCacheKey()` but in Coil's website this is used for regular views, not Compose. And even if it can be used with Compose it would be much easier to support the placeholder attribute directly.

**Comment by saket** [comment]
Good point, let's add parameters for `placeholder`, `error` and `fallback`. Wanna send a PR?

**Comment by MV-GH** [comment]
You can get all these already by setting them on the imageLoader and the ImageRequest

**Comment by saket** [comment]
Yea you can indeed set placeholders through `ImageRequest.Builder` but it only accepts drawables and resources. You can't use painters through telephoto yet.

@minas1 fwiw `placeholderMemoryCacheKey` is different from `placeholder`. It is useful for creating shared element transitions by using a previously loaded image in lower quality as the placeholder for loading a larger resolution variant.

**Comment by muriloonunes** [comment]
Hey, any updates on this? I also need both `placeholder` and `error` parameters
